xrpl/transaction/batch_signers.py

```
@@ -0,0 +1,134 @@
+            )
+        if (
+            tx.signing_pub_key != ""
+            or tx.txn_signature is not None
+            or tx.signers is not None
+        ):
+            raise XRPLException("Transaction must be unsigned.")
+
+    batch_txs = cast(List[Batch], decoded_txs)
+    _validate_batch_equivalence(batch_txs)
+
+    return encode(_get_batch_with_all_signers(batch_txs).to_xrpl())
```

Pylint (3.3.7)

[error] Instance of 'BaseModel' has no 'to_xrpl' member

(E1101)